DxDispatch/src/dxdispatch/main.cpp

```diff
@@ -70,8 +70,11 @@ int main(int argc, char** argv)
         try
         {
             Adapter adapter = Adapter::Select(dxCoreModule, args.AdapterSubstring());
+            LogInfo(adapter.GetDetailedDescription() + "\n");
```

Comment:
Final thought: we may not want to print this every time the exe runs. You can get this by adding `-s` option, and some folks might be relying on the current output format for parsing in scripts.